.github/workflows/ci.yml

```diff
@@ -130,7 +130,7 @@ jobs:
           --health-timeout 5s
           --health-retries 5
       mongodb:
-        image: mongo:5.0.11
+        image: ${{ (matrix.ruby-version == '2.2.10' || matrix.ruby-version == '2.3.8' || matrix.ruby-version == '2.4.10') && 'mongo:5.0.11' || 'mongo:latest' }}
```

Comment:
Can we stuff the 3 values in an array?

```javascript
['2.2.10', '2.3.8', '2.4.10'].includes(matrix.ruby-version) ? 'mongo:5.0.11' : 'mongo:latest'
```

Reply:
Great suggestion! Refactor riff in: eea71a7245859b18fc212c3f78a23e016b814c4f